stats/gRPC.md

```diff
@@ -9,117 +9,124 @@ Units are encoded according to the case-sensitive abbreviations from the [Unifie
 
 * Latencies are measures in float64 milliseconds, denoted "ms"
 * Sizes are measured in bytes, denoted "By"
-* Dimensionless values have unit "1"
+* Counts have unit "1"
 
 Buckets for distributions in default views are as follows:
 
 * Size in bytes: 0, 1024, 2048, 4096, 16384, 65536, 262144, 1048576, 4194304, 16777216, 67108864, 268435456, 1073741824, 4294967296
 * Latency in ms: 0, 0.01, 0.05, 0.1, 0.3, 0.6, 0.8, 1, 2, 3, 4, 5, 6, 8, 10, 13, 16, 20, 25, 30, 40, 50, 65, 80, 100, 130, 160, 200, 250, 300, 400, 500, 650, 800, 1000, 2000, 5000, 10000, 20000, 50000, 100000
 * Counts (no unit): 0, 1, 2, 4, 8, 16, 32, 64, 128, 256, 512, 1024, 2048, 4096, 8192, 16384, 32768, 65536
```

Comment:
Should we mention that these are for the [sent, received]_messages_per_rpc?

Reply:
Yes please, I'd like it to be really clear what "qps" is as a unit. :)

Reply:
MHz?

Reply:
None of the units here have the "/sec" divisor. Count is count until my monitoring system calculates the rate over time?

Reply:
Correct, we produce cumulative data not rates. it is easy for backends to compute rates out of cumulative data.